.env.example

```
@@ -7,10 +7,12 @@ APP_BUCKET_NAME=dev-sg-dashboard
 APP_FOLDER_NAME=sg-dashboard 
 
 # Firebase Configuration 
-APP_API_KEY=your_api_key_here 
-APP_AUTH_DOMAIN=your_auth_domain_here 
-APP_PROJECT_ID=your_project_id_here 
-APP_STORAGE_BUCKET=your_storage_bucket_here 
-APP_MESSAGE_SENDER_ID=your_messaging_sender_id_here 
-APP_APP_ID=your_app_id_here 
-APP_MEASUREMENT_ID=your_measurement_id_here give text sample key
\ No newline at end of file
+APP_API_KEY="your_api_key_here" 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)